docs/source/faq/index.rst

```diff
@@ -49,6 +49,33 @@ App will launch/update in the output of your notebook cells!
 Check out the :ref:`notebook environment guide <notebooks>` for more
 information about running FiftyOne in notebooks.
 
+.. _faq-remote-notebook-support:
+
+Can I use FiftyOne in a remote notebook?
+----------------------------------------
+
+Yes! A common notebook workflow is to `launch jupyter on a remote system and
+connect to it from a local browser 
+<https://ljvmiranda921.github.io/notebook/2018/01/31/running-a-jupyter-notebook>`_.
+
+If you want to use the FiftyOne App in this notebook, you will also need to
+port forward FiftyOne from the remote machine to your local machine with the
+following.
+
+**You need to use the same port for FiftyOne on your remote and local machine.**
```

Comment:
Don't you have to use port 5151 exactly?

If the user runs `fo.launch_app(dataset)`, this will default to port 5151 on the remote machine, which is then forwarded to the local machine in this case.

Reply:
You're right, I just tested it with a different port and it didn't work. Fixed